### PR TITLE
Remove error word from info logs *40140

### PR DESCRIPTION
**Pull request by tolya95x106**
Story reference: https://tickets.vivino.com/issues/40140

## How did you implement this?

Moved initialization of LogFormatter to cfgrevel.

### CODE REVIEW

File: logger.go [hunk 1]
```diff
@@ -100,7 +102,21 @@ var LogFormatter = func(values ...interface{}) (messages []interface{}) {
 			messages = append(messages, fmt.Sprintf(" \n\033[36;31m[%v]\033[0m ", strconv.FormatInt(values[5].(int64), 10)+" rows affected or returned "))
 		} else {
 			messages = append(messages, "\033[31;1m")
-			messages = append(messages, values[2:]...)
+
+			var message interface{}
```
Comment (eikmadsen) [inline logger.go]: The entire gorm package is database agnostic (until now) ... 
I suggest you revert your changes here, and instead override the LogFormatter in cfgrevel package.

**Review by eikmadsen — CHANGES_REQUESTED** [review]

**Review by eikmadsen — CHANGES_REQUESTED** [review]
No need to delete all the existing code. Just override it in cfgrevel as you are doing now ;-)

**Review by eikmadsen — CHANGES_REQUESTED** [review]
Why is this in the GORM fork ? It could a function in go-tools or go-common instead.  
The LogFormatter var is exported so you can override it locally where you need it

**Comment by tolya95x106** [comment]
This is refactoring and no new functionality.

**Comment by eikmadsen** [comment]
I see, difficult to tell from the changes without doing a very thorough review. 
Could we skip the refactor ? If you are setting the LogFormatter, then all we get from these changes are potential merge conflicts when upgrading gorm.

**Pull request closed without merging**